package-lock.json

```diff
@@ -0,0 +1,3381 @@
+{
```

Comment:
While I agree that having a package-lock.json file is a good practice, I don't think it relates to the stated goal of the PR.